servicepulse/intro-failed-messages.md

```diff
@@ -55,7 +58,7 @@ If a message fails repeated retry attempts, an indication is added, including th
   
 ![Repeated failure indication](images/failed-messages-repeated-failure.png)
 
-NOTE: Tracking the number of repeated retry attempts may be significant to your specific solution implementation, since each retry may invoke custom and/or third party logic that may not participate in the NServiceBus endpoint transactional  processing (and is therefore not rolled back on processing failure).
+NOTE: Tracking the number of repeated retry attempts may be significant, since each retry may invoke custom and/or third party logic that may not participate in the NServiceBus endpoint transactional  processing (and is therefore not rolled back on processing failure).
```

Comment:
maybe...to be honest I only have a vague idea of what its trying to say. It kind of implies 'significant' is a metric for  a reason that looks to be unrelated to the users intent.